app/controllers/users_controller.rb

```diff
@@ -1,2 +1,12 @@
 class UsersController < ApplicationController
+  before_action :authorize_user
```

Comment:
This is there intentionally. I don't think we want anyone who isn't authorized to be able to see a user list.